imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -123,5 +197,49 @@ func TestBootTime(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to convet start time %s", metadata)
 	}
-	t.Logf("image boot time is %d", time.Now().Second()-startTime)
+	t.Logf("image start time is %d", time.Now().Second()-startTime)
+}
+
+func TestBootTime(t *testing.T) {
+	image, err := utils.GetMetadata("image")
+	if err != nil {
+		t.Fatalf("couldn't get image from metadata")
+	}
+
+	MAX_THRESHOLD := getThresholdValue(image)
```

Comment:
why not maxThreashold instead of MAX_THREASHOLD?

Reply:
Done